bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -114,8 +114,8 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
   SeqNum maxSeqNumTransferredFromPrevViews = 0;
 
   // requests queue (used by the primary)
-  std::queue<ClientRequestMsg*> requestsQueueOfPrimary;  // only used by the primary
-  size_t primaryCombinedReqSize = 0;                     // only used by the primary
+  std::queue<std::unique_ptr<ClientRequestMsg>> requestsQueueOfPrimary;  // only used by the primary
+  size_t primaryCombinedReqSize = 0;                                     // only used by the primary
 
   std::map<uint64_t, std::pair<Time, ClientRequestMsg*>>
       requestsOfNonPrimary;  // used to retransmit client requests by a non primary replica
```

Comment:
reminder about this one: add a new task since it holds a raw pointer and was used in your chain of calls.

Reply:
https://jira.eng.vmware.com/browse/BC-26934